habitat_baselines/utils/common.py

```diff
@@ -74,6 +75,56 @@ def forward(self, x: Tensor) -> CustomFixedCategorical:
         return CustomFixedCategorical(logits=x)
 
 
+class CustomNormal(torch.distributions.normal.Normal):
+    def sample(
+        self, sample_shape: Size = torch.Size()  # noqa: B008
+    ) -> Tensor:
+        return super().rsample(sample_shape)
+
+    def log_probs(self, actions):
+        ret = super().log_prob(actions).sum(-1).unsqueeze(-1)
+        return ret
+
+
+class GaussianNet(nn.Module):
+    def __init__(
+        self,
+        num_inputs: int,
+        num_outputs: int,
+        config: Config,
+    ) -> None:
+        super().__init__()
+
+        self.config = config
+        if config.use_log_std:
+            self.min_std = config.min_log_std
+            self.max_std = config.max_log_std
+        else:
+            self.min_std = config.min_std
+            self.max_std = config.max_std
+
+        self.mu = nn.Linear(num_inputs, num_outputs)
+        self.std = nn.Linear(num_inputs, num_outputs)
+
+        nn.init.orthogonal_(self.mu.weight, gain=0.01)
+        nn.init.constant_(self.mu.bias, 0)
+        nn.init.orthogonal_(self.std.weight, gain=0.01)
+        nn.init.constant_(self.std.bias, 0)
+
+    def forward(self, x: Tensor) -> CustomNormal:
+        mu = self.mu(x)
+        if self.config.action_activation == "tanh":
```

Comment:
Can you encapsulate this logic in `__init__` -- that'll work with TorchScript which can be worth doing for some models.

i.e.
```
self.mu_act = ....
```

Reply:
Done